### ISSUE TITLE
remove this repo

### ISSUE THREAD

**Issue by dacrome** [osiam/pkg-jackson-databind#1]
Do we need this repo anymore? I think this was created because of license issues, right?

**Comment by tkrille** [comment]
dunno ... +1 for removing

**Comment by mirabilos** [comment]
Wrong. This repository was created for Debian packaging, because it’s a dependency of OSIAM.

There is currently a newer version of jackson-{core,annotations,databind} in Debian; therefore, this repository is obsolete and may be deleted.

**Comment by dacrome** [comment]
ok
